odoo_backup_sh/__init__.py

```diff
@@ -2,3 +2,15 @@
 
 from . import controllers
 from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def uninstall_hook(cr, registry):
+    # During the unistall of the module, we have the error:
+    # 'update or delete on table "ir_act_server" violates foreign key constraint
+    # "ir_cron_ir_actions_server_id_fkey" on table "ir_cron"'. Due to this error the "odoo_backup_sh.config" table
```

Comment:
Would be better if you explain why errors is raised. It not a good idea to force eacg reader think about it.

The rease as I understand it: "odoo_backup_sh.config.cron" inherits from "ir.cron", which inherits from "ir.action.server", which as field model_id with option ondelete='cascade'. During uninstallation model is removed, so odoo tries to delete corresponding ir.action.server records which which raise violations error, because those records are used in "ir.cron" records. So, we need to delete "odoo_backup_sh.config.cron manually